qiskit_optimization/algorithms/multistart_optimizer.py

```diff
@@ -112,7 +113,8 @@ def trials(self, trials: int) -> None:
         Args:
             trials: The number of trials to set.
         """
-        self._trials = trials
+        if trials > 0:
```

Comment:
Could you raise a ValueError if `trials <= 0` as same as https://github.com/Qiskit/qiskit-optimization/pull/110/files#r622521214?

Reply:
Yeah sure.